DancingGoat/Views/Shared/_Layout.cshtml

```diff
@@ -13,9 +11,10 @@
 <head id="head">
     <!-- Google Tag Manager -->
     <script>
-        (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
-        var f = d.getElementsByTagName(s)[0], j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : '';
-        j.async = true; j.src = 'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+        (function (w, d, s, l, i) {
```

Comment:
this code block may not be changed

Reply:
It's just 'prettified' by VS, it works and does the same thing as before. But I've reverted it to original gtm script.